### PR TITLE
Fix Latex build of docs

### PR DESCRIPTION
**Pull request by lmoureaux**
Supersedes #1831.

### CODE REVIEW

**Review by jwrober — APPROVED** [review]
I'd like to see the one formula broken up into two lines. Rest is fine.

File: docs/Coding/ai.rst [hunk 1]
```diff
@@ -97,25 +97,23 @@ Estimation of Profit From a Military Operation
...
-
-Here :math:`\texttt{Unhappiness_Compensation}` is from a military unit being away from home and
+Here :math:`\texttt{Unhappiness\_Compensation}` is from a military unit being away from home and
 :math:`\texttt{Support}` is the number of Shields spent on supporting this unit per turn.
 
-:math:`\texttt{Battle_Profit} = (\texttt{Shields_Lost}_\texttt{enemy} \times \texttt{Probability}_\texttt{win})
-- (\texttt{Shields_Lost}_\texttt{us} \times \texttt{Probability}_\texttt{lose})`
+.. math::
+  \texttt{Battle\_Profit} = (\texttt{Shields\_Lost}_\texttt{enemy} \times \texttt{Probability}_\texttt{win})
+                           - (\texttt{Shields\_Lost}_\texttt{us} \times \texttt{Probability}_\texttt{lose})
```
[the remark this answers is not in the document]
Comment (lmoureaux) [reply docs/Coding/ai.rst:114]: For sure. I didn't check the HTML output but this looks pretty bad with the \. What is MathJax doing? :o

[the remark this answers is not in the document]
Comment (jwrober) [reply docs/Coding/ai.rst:114]: This also reinforces the need for #1719

[the remark this answers is not in the document]
Comment (lmoureaux) [reply docs/Coding/ai.rst]: Should look better now

**Review by jwrober — APPROVED** [review]
LGTM

**Pull request merged**